v2/jwk/jwk.go

```diff
@@ -105,3 +124,19 @@ func RefreshKeySets() (err error) {
 
 	return
 }
+
+func getKeySetsURL() (string, error) {
+	if config == nil && KeySetURL == "" {
+		return "", errors.New("jwk is not configured")
+	}
+
+	if config == nil {
+		return KeySetURL, nil
+	}
+
+	if config.Stage == stages.StageProd {
```

Comment:
You are not handling stageLocal here?

Reply:
I was thinking about that as well, what value do you see?

Reply:
I'm not sure how this is used. But I think that if we have localStage as part of the stage"enum" it should be handled here.?

Reply:
I added the `StageLocal` as a constant because it is used in the services.

The reason why I decided not to handle `StageLocal` in the packages: `jwk` and `auth` was mainly because I wasn't sure I wanted to configure it like that and due to the added complexity of the configuration, since you would need to handle unique ports and stages for different services.